Quotient/room.cpp

```diff
@@ -359,7 +361,7 @@ class Q_DECL_HIDDEN Room::Private {
 
     QJsonObject toJson() const;
 
-    bool isLocalUser(const User* u) const { return u == q->localUser(); }
+    bool isLocalMember(const QString& memberId) const { return q->member(memberId) == q->localMember(); }
```

Comment:
```suggestion
    bool isLocalMember(const QString& memberId) const { return memberId == connection->userId(); }
```
would do the same thing, I suppose?